bips/obpp-03.mediawiki

```diff
@@ -71,10 +73,27 @@ When one of the following cases exist, a compliant implementation may create an
 ## Add a spend output to the transaction.
 ## Add a change output to the transaction containing the difference between the current set of inputs and the desired spend.
 # Repeat step 1 to create a second spend output and change output.
+# Repeat step 2 zero or more times.
```

Comment:
Should this BIP recommend a method of choosing the number of times to repeat this step?